scripts/icon-sprite/make-sprite.ts

```diff
@@ -40,9 +42,15 @@ async function makeSprite(): Promise<void> {
 
     fs.writeFileSync(targetPath, buffer);
 
+    const hash = crypto.createHash('md5').update(buffer).digest('hex').slice(0, 10);
```

Comment:
Any specific reason to prefer `crypto` over `uuid`?

Reply:
@denysbutenko 

Using UUID from angular2-uuid will indeed generate a unique identifier, but it will not be content-dependent. This means every time you build, it will produce a different value, regardless of whether the content has changed or not. This approach is less ideal for caching purposes, as it will effectively disable caching by creating a new URL each time, even if the sprite.svg content remains unchanged.

Reply:
Okay thanks for explaining